src/SDK/MonitoringApplianceEmulator/Controls/HypertextBlock.cs

```diff
@@ -0,0 +1,135 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="HypertextBlock.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.Azure.Monitoring.SmartDetectors.MonitoringApplianceEmulator.Controls
+{
+    using System;
+    using System.Diagnostics;
+    using System.Text.RegularExpressions;
+    using System.Windows;
+    using System.Windows.Controls;
+    using System.Windows.Documents;
+
+    /// <summary>
+    /// An extension of <see cref="TextBlock"/> control that displays hypertext. The hypertext should be transfered using the <see cref="Hypertext"/> dependency property.
+    /// </summary>
+    public class HypertextBlock : TextBlock
+    {
+        /// <summary>
+        /// The hypertext dependency property. This text may include multiple hyperlinks in valid HTML A tags.
+        /// For example, '<a href="https://msdn.microsoft.com">Developer Network</a>' will be replaced with a hyperlink with display text of "Developers Network".
+        /// </summary>
+        public static readonly DependencyProperty HypertextProperty = DependencyProperty.Register(
+            "Hypertext",
+            typeof(string),
+            typeof(HypertextBlock),
+            new FrameworkPropertyMetadata(
+                string.Empty,
+                new PropertyChangedCallback(OnHypertextPropertyChanged)),
+            HypertextValidateCallback);
+
+        private static readonly Regex HtmlHyperlinkRegex = new Regex("<a [^>]*href[ ]*=[ ]*(?<href>(?:'.*?')|(?:\".*?\"))[ ]*>(?<linkText>[^<]*)</a>");
```

Comment:
>[ ]*=[ ]* [](start = 82, length = 9)

Prefer using "\s*" instead of "[ ]*" - it will match any whitespace (such as tab, etc.), not just space